moduleframework/mtf_init.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+# Script generates super easy template of test for module docker
+# Purpose of this script is to generate needed files to start testing
+# Author Petr Sklenar psklenar@gmail.com
+#
+# Meta test family (MTF) is a tool to test components of a modular Fedora:
+# https://docs.pagure.org/modularity/
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+
+from optparse import OptionParser
+import os.path
+
+def get_options():
+    parser = OptionParser(usage="usage: %prog [options]",
+                          version="%prog 1.0")
+    parser.add_option("-n", "--name",
+                      default="name",
+                      dest="name",
+                      help="Name of module for testing")
+    parser.add_option("-c", "--container",
+                      dest="container",
+                      default="docker.io/modularitycontainers/memcached",
+                      action="store",
+                      help="Specify container path, example: docker.io/modularitycontainers/memcached")
+    (options, args) = parser.parse_args()
+
+    return options
+
+class Template(object):
+    def __init__(self, name, container):
+        self.name = name
+        self.container = container
+
+    def set_content_config_yaml(self):
+        self.filePathConfig = 'config.yaml'
+        configYaml = """# this is generated config.yaml with minimum stuff
+---
+document: meta-test
+version: 1
+name: {name}
+default_module: docker
+module:
+    docker:
+        container: {container}
+""".format(name=self.name, container=self.container)
+        self.configYaml = configYaml
+
+    def set_content_test_py(self):
+        self.filePathTest = 'test.py'
+        test = '''#!/usr/bin/python
```

Comment:
What about to have the script in https://github.com/fedora-modularity/meta-test-family/tree/devel/examples/template